index.html

```diff
@@ -464,6 +465,51 @@ <h2>
       Integration with Credential Management API
     </h2>
     <aside class="issue" data-number="65"></aside>
+    <h3>
+      The `<dfn>clientAllowsProtocol()</dfn>` method
+    </h3>
+    <p>
+      The {{DigitalCredential/clientAllowsProtocol()}} method allows digital
+      credential verifiers and issuers to determine which 
+      [=digital credential/exchange protocols=] the client supports for
+      interactions with [=credential providers=].
+    </p>
+    <p class="note">
+      This method does not convey [=digital credential/request protocol=] or 
+      [=digital credential/issuance protocol=] support in the [=client device=]
+      or [=credential provider=].
+    </p>
+    <p>
+        User agents SHOULD NOT vary the response value based on any information
+        about availability of hardware, presence or configuration of software,
+        wallets, credential providers, or digital credentials, or user
+        configuration or preferences. If the response value varied, the user
+        agent would introduce risks both of fingerprinting and of silently
+        revealing other details about user behavior or configuration. The
+        response value SHOULD vary only by user agent major version and
+        indicate whether the browser supports distributing requests with that
+        protocol to underlying platform or provider.
+    </p>
+    <h4>
+      clientAllowsProtocol algorithm
+    </h4>
+    <p>
+      When this method is invoked, the user agent MUST execute the following
+      algorithm:
+    </p>
+    <ol class="algorithm">
+      <li>
+        If |protocol| contains characters other than lowercase ASCII letters,
+        digits, or hypens, throw a "{{ConstraintError}}" {{DOMException}}.
+      </li>
+      <li>
+        If |protocol| is known and allowed by the user agent, 
+        return <code>true</code>.
```

Comment:
To be "allowed" implies that it is known, so just: 

```suggestion
        If |protocol| allowed by the user agent, 
        return <code>true</code>.
```